docs/example-scenario/ai/movie-recommendations.md

```diff
@@ -0,0 +1,117 @@
+4. Azure Machine Learning service is used for coordinating the experimentation, such as hyperparameter sweeping and model management.
+5. A trained model is preserved on Azure Cosmos DB, which can then be applied for recommending the top *k* movies for a given user.
+6. The model is then deployed onto a web or app service by using Azure Container Instances or Azure Kubernetes Service.
+
+### Components
+
+* [Data Science Virtual Machine][dsvm] (DSVM) is an Azure virtual machine with deep learning frameworks and tools for machine learning and data science. The DSVM has a standalone Spark environment that can be used to run ALS.
+* [Azure Blob storage][blob] stores the dataset for movie recommendations.
+* [Azure Machine Learning service][mls] is used to accelerate the building, managing, and deploying of machine learning models.
+* [Azure Cosmos DB][cosmosdb] enables globally distributed and multi-model database storage.
+* [Azure Container Instances][aci](https://docs.microsoft.com/en-us/azure/container-instances/container-instances-overview) is used to deploy the trained models to web or app services, optionally using [Azure Kubernetes Service][aks].
+
+### Alternatives
+
+[Azure Databricks][databricks] is a managed Spark cluster where model training and evaluating is performed. You can set up a managed Spark environment in minutes, and [autoscale][autoscale] up and down to help reduce the resources and costs associated with scaling clusters manually. Another resource-saving option is to configure inactive [clusters][clusters] to terminate automatically.
+
+## Considerations
+
+### Availability
+
+The DSVM is available in [multiple regions][regions]  around the globe and meets the [service level agreement][sla] (SLA) for virtual machines.
+
+The [availability checklist][availability] in the Azure Architecture Center lists more best practices.
+
+### Scalability
+
+You can scale a VM up or down by changing the [VM size][vm-size]. To scale out horizontally, put two or more VMs behind a load balancer as in a typical [n-tier architecture][n-tier].
+
+### Security
+
+This scenario uses [Azure Active Directory B2C][aad] (Business 2 Consumer) to authenticate users. For general guidance on designing secure solutions, see the [Azure Security Documentation][sec-docs].
+
+### Resiliency
+
+All the components in this scenario are managed, making them resilient at a regional level. For general guidance on designing resilient solutions, see [Designing resilient applications for Azure][resiliency].
+
+## Deploy this scenario
+
```

Comment:
Let's include a "Deploy to Azure" button that links to the Github repo here to make it more visable.

`<a
href="https://github.com/Microsoft/Recommenders" target="_blank">
    <img src="https://azuredeploy.net/deploybutton.png"/>
</a>`

Reply:
OK